docs/integrations/sources/workday-raas.md

```diff
@@ -0,0 +1,38 @@
+# Workday RAAS
```

Comment:
With the new enterprise connector docs system, we no longer need this file to be here